ci-templates/stages.yml

```diff
@@ -186,7 +186,9 @@ stages:
         parameters:
           poolSettings: ${{parameters.poolSettings}}
           expectUseVMs: ${{parameters.expectUseVMs}}
-          modules: main
+          modules:
+            - main
+            - quartz
```

Comment:
I don't think we want to add an extension to the `main` one. Let's put it elsewhere (and reorganize if needed).